dockerfiles/Dockerfile.build

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:11
+LABEL maintainer="Digirati <opensource@digirati.com>"
+
+USER root
+
+RUN apt-get clean && apt-get update && apt-get -y upgrade \
+    && apt-get install -y python python-pip software-properties-common lsb-release apt-transport-https \
+    && curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - \
+    && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable" \
+    && apt-get update && apt-get -y install docker-ce
+
+RUN pip install pre-commit
+
+USER gradle
```

Comment:
We're switching to the `gradle` user before it's actually been created. This doesn't cause an error during `docker build`, but any attempt to actually use this user will result in `unable to find user gradle: no matching entries in passwd file`.